test/testtask_test.rb

```diff
@@ -3,18 +3,28 @@
 require 'rake/testtask'
 
 class TestTaskTest < Minitest::Test
-  def test_attributes
-    rake_task = nil
+  def setup
+    @name = 'sample_name'
+    @rake_task = nil
     # This will be in users' Rakefiles
-    Rake::TestTask.new { |t|
-      t.libs << "test"
+    @ttnt_task = Rake::TestTask.new { |t|
```

Comment:
I guess this is defined at the wrong place, you should rather do `@ttnt_task = TTNT::TestTask.new(t)` line 14.